README.md

```diff
@@ -23,10 +23,10 @@ If you use [Maven](http://maven.apache.org/), you can add Pushy to your project
 
 If you don't use Maven (or something else that understands Maven dependencies, like Gradle), you can [download Pushy as a `.jar` file](https://github.com/relayrides/pushy/releases/download/pushy-0.10/pushy-0.10.jar) and add it to your project directly. You'll also need to make sure you have Pushy's runtime dependencies on your classpath. They are:
 
-- [netty 4.1.12](http://netty.io/)
+- [netty 4.1.12.Final](http://netty.io/)
```

Comment:
Leaving the "final" out was actually a deliberate choice here. I absolutely agree that it's helpful for finding the right version in Maven Central, but (almost) by definition, the folks who are doing this manually aren't using Central. For human readability, I think it's best to just stick with the version number.